ovmf-vars-generator

```diff
@@ -28,12 +28,24 @@ def strip_special(line):
 
 
 def generate_qemu_cmd(args, readonly, *extra_args):
-    if args.disable_smm:
+    is_x86 = True
+    arch_args = []
+    if os.path.basename(args.qemu_binary) == 'qemu-system-aarch64':
```

Comment:
I'd suggest keying off of `args.arch` instead. Say if someone were to save off a copy of `qemu-system-aarch64` to `qemu-system-aarch64.working` and re-run with that - a change in behavior would seem unexpected.